src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -59,6 +59,10 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
     @Optional
     Boolean pull
 
+    @Input
+    @Optional
+    Map<String, String> buildArgs
```

Comment:
Lets init this to an empty map: `... buildArgs = [:]`. I generally like to add (personal preference) instead of passing in a new map. Init'ing to an empty map will allow both.